src/Tribe/Organizer.php

```diff
@@ -640,7 +640,12 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$organizer_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventOrganizerID' ) );
+			$organizer_ids = array_filter(
+				array_map(
+					'absint',
+					(array)get_post_meta( $event, '_EventOrganizerID' )
```

Comment:
Space after `(array)`
```suggestion
					(array) get_post_meta( $event, '_EventOrganizerID' )
```